src/test/resources/processor/given_entity.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
instead of an arbitrary json, it would be good to use the real instance

Reply:
This entity is just an arbitrary JSON to show various kinds of rules for the RuleProcessorTest :)   The real instance is in the _given_inventory_instance.json_ file, see comment above :)